api/base/serializers.py

```diff
@@ -1252,7 +1252,10 @@ def to_representation(self, obj, envelope='data'):
             else:
                 try:
                     if hasattr(attribute, 'all'):
-                        representation = field.to_representation(attribute.all())
+                        if field.field_name == 'tags':
```

Comment:
This doesn't really belong here. Can we figure out a way to move this logic onto the field? Maybe we have a special field

```python
class ValuesListField(Field):
    # ...

    def to_representation(self, val):
        return val.values_list(**self.values_list_kwargs)

tags = ValuesListField('name', flat=True)
```